code/modules/item_worth/reagents.dm

```diff
@@ -757,4 +757,6 @@
 	value = 500
 
 /datum/reagent/toxin/undead
-	value = 300
\ No newline at end of file
+	value = 300
+/datum/reagent/alcohol/ethanol/rumandcola
```

Comment:
This should go with the rest of the ethanol datums up there.

Reply:
I wasn’t sure if I should insert it there or at the end.  I’ll take a look at resolving this when I get home